packages/ember/package.json

```diff
@@ -85,7 +85,7 @@
     "node": "10.* || >= 12"
   },
   "ember": {
-    "edition": "octane"
+    "edition": "classic"
```

Comment:
I didn't make this change, so I'm guessing maybe the test runner did? I have no idea if it's a good one or not. Happy to revert if not.

Reply:
I have no clue either :3

Reply:
@k-fish - Any chance you can weigh in here?

Reply:
Yeah you should leave this as octane unless there are issues, octane is the latest edition and this addon should support it.

Reply:
Reverted